site/docs/serialization/basics.md

```diff
@@ -0,0 +1,26 @@
+# Basics
+
+Substrait is designed to be serialized into various different formats. Currently we support a binary serialization for
+transmission of plans between programs (e.g. IPC or network communication) and a text serialization for debugging and
+human readability. Other formats may be added in the future.
+
+These formats serialize a collection of plans. Substrait does not define how a collection of plans is to be interpreted.
+For example, the following scenarios are all valid uses of a collection of plans:
+
+- A query engine receives a plan and executes it. It receives a collection of plans with a single root plan. The
+  top-level node of the root plan defines the output of the query. Non-root plans may be included as common subplans
+  which are referenced from the root plan.
+- A transpiler may convert plans from one dialect to another. It could take, as input, a single root plan. Then
+  it could output a serialized binary containing mulitple root plans. Each root plan is a representation of the
```

Comment:
Type: `multiple`.

```suggestion
  it could output a serialized binary containing multiple root plans. Each root plan is a representation of the
```